### PR TITLE
fix system table replica doc

### PR DESCRIPTION
**Pull request by AilinKid**
Signed-off-by: AilinKid <314806019@qq.com>

### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tiflash/use-tiflash.md [hunk 1]
```diff
@@ -60,6 +60,9 @@ ALTER TABLE `tpch50`.`lineitem` SET TIFLASH REPLICA 0
 
 * 不推荐同步 1000 张以上的表，这会降低 PD 的调度性能。这个限制将在后续版本去除。
 
+注意
+> v5.1 版本及其后续版本将不再支持系统表的 replica 设置，在集群升级之前需要将相关系统表的 replica 清除，不然升级到高版本时候将无法再修改系统表的 replica 设置。
```
Comment (TomShawn) [inline tiflash/use-tiflash.md:64]:
```suggestion
+ v5.1 版本及后续版本将不再支持设置系统表的 replica。在集群升级前，需要清除相关系统表的 replica，否则升级到较高版本后将无法再修改系统表的 replica 设置。
```

**Review by zimulala — COMMENTED** [review]
LGTM

**Comment by AilinKid** [comment]
@Liuxiaozhen12 should we merge it now?

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**